src/client.jl

```diff
@@ -375,6 +382,43 @@ function run_remote_repl_command(conn, out_stream, cmdstr)
     return result_for_display
 end
 
+"""
+    remotecmd(cmdstr::String)
+
+Evaluate `cmdstr` in the last opened RemoteREPL connection and print result to `Base.stdout`
+"""
+remotecmd(cmd::String) = remotecmd(_repl_client_connection, Base.stdout, cmd)
+
+"""
+    remotecmd(conn::Connection, cmdstr::String)
+
+Evaluate `cmdstr` in the connection `conn` and print result to `Base.stdout`.
+"""
+remotecmd(conn::Connection, cmd::String) = remotecmd(conn, Base.stdout, cmd)
+
+"""
+    remote_module!(conn::Connection = _repl_client_connection, mod::Module)
+
+Change future remote commands in the session of connection `conn` to be evaluated into module `mod`.
+The default connection `_repl_client_connection` is the last established RemoteREPL connection.
+If the module cannot be evaluated locally pass the name as a string.
+Equivalent to using the `%module` magic.
+"""
+remote_module!(conn::Connection, mod::Module) = remotecmd(conn, Base.stdout, "%module $(mod)")
+remote_module!(mod::Module) = remotecmd(_repl_client_connection, Base.stdout, "%module $(mod)")
```

Comment:
Ideally these would not use string concatenation because the `%module` just needs to be parsed again on the client side in order to send the `:in_module` command. But doing that cleanly that would require a bit of refactoring to pull out the last couple of paragraphs of `remotecmd()` into a separate internal function dedicated to sending already-parsed command tuples. So I leave it to you to decide what to do about this.